src/ui/map.js

```diff
@@ -793,9 +791,8 @@ class Map extends Camera {
      * If the map's current pitch is lower than the new minimum,
      * the map will pitch to the new minimum.
      *
-     * @param {number | null | undefined} minPitch The minimum pitch to set (0-85).
-     *   If `null` or `undefined` is provided, the function removes the current minimum pitch (i.e. sets it to 0).
-     * @returns {Map} `this`
+     * @param {number | null | undefined} minPitch The minimum pitch to set (0-85). If `null` or `undefined` is provided, the function removes the current minimum pitch and resets it to 0.
```

Comment:
```suggestion
     * @param {number | null | undefined} `minPitch` The minimum pitch to set (0-85). If `null` or `undefined` is provided, the function removes the current minimum pitch and resets it to 0.
```

Reply:
In other examples that name the parameter, it doesn't appear to be in back ticks

Reply:
Yes you're right I read this too fast, it's the name of the parameter, all good here.